packages/good-design/src/advanced/web3action/Web3Action.tsx

```diff
@@ -25,11 +26,11 @@ const throwCancelled = (e: any) => {
   }
 };
 
-const StepIndicator = ({ text }: { text?: string | undefined }) => {
+const StepIndicator: FC<{ text?: string } & ITextProps> = ({ text, color = "primary.500", fontSize = "md" }) => {
```

Comment:
wrong

1. should be configured in .theme file
2. font sizes and color names should be configured in global theme (by @alt-a1r )